frontend/views/components/GroupSettings/InviteActionButton.vue

```diff
@@ -0,0 +1,58 @@
+<template lang="pug">
+menu-parent
+  menu-trigger.is-icon
+    i.icon-ellipsis-v
+
+  menu-content.c-content
+    ul
+      menu-item(
+        tag='router-link'
+        item-id='hash-3'
+        to='/somewhere'
+      )
+        i.icon-vote-yea.c-icon
+        | See original proposal
+      menu-item(
+        tag='router-link'
+        item-id='hash-3'
+        v-if='!linkExpired'
+        to='/somewhere'
+      )
+        i.icon-times.c-icon
+        | Revoke link
+</template>
+
+<script>
+import { MenuParent, MenuTrigger, MenuContent, MenuItem } from '@components/Menu/index.js'
+
+export default {
+  name: 'inviteActionButton',
```

Comment:
i think this file should be called `InviteLinksActionButton` to follow the same name as the other file. the same for the component name.